ui/lib/kmip/addon/components/edit-form-kmip-role.js

```diff
@@ -0,0 +1,53 @@
+import EditForm from 'core/components/edit-form';
+import layout from '../templates/components/edit-form-kmip-role';
+import { Promise } from 'rsvp';
+
+export default EditForm.extend({
+  layout,
+  display: null,
+  init() {
+    this._super(...arguments);
+    let display = 'operationAll';
+    if (this.model.operationNone) {
+      display = 'operationNone';
+    }
+    if (!this.model.isNew && !this.model.operationNone && !this.model.operationAll) {
+      display = 'choose';
+    }
+    this.set('display', display);
+  },
+
+  actions: {
+    updateModel(val) {
+      if (val === 'operationAll') {
+        this.model.set('operationNone', false);
```

Comment:
a comment explaining that setting these will automatically “clear” the other fields might be helpful here

Reply:
Ah yep, this is for toggling the radio group, so it won't clear the other fields until you save so you can go through all, none, choose and not lose the list you've picked in "choose" (this is all a long explanation that I will add as a comment 😄).